examples/agentInteractionsDemo/app.js

```diff
@@ -202,25 +245,18 @@ new Vue({
                     .then((data) => {
                         // Process evaluations data
                         const evaluations = data.entities;
-                        evaluations.forEach((eval) => {
-                            conversationsApi.getConversation(eval.conversation.id)
-                                .then((conv) => {
-                                    let newConv = conv;
-                                    newConv.customer = conv.participants.find((part) => {
-                                        return part.purpose === "customer";
-                                    });
-                                    this.conversationsData.conversations.push(newConv);
+                        getEvalConversationsData(evaluations)
+                            .then((evalConversations) => {
+                                evalConversations.forEach((evalConv, index) => {
+                                    this.conversationsData.conversations.push(evalConv);
```

Comment:
Since `getEvalConversationsData` catches errors and returns `undefined`, you'll want to filter out falsey evalConv instances.  It's important to just return here as the index is used to match the conversation with the evaluations.

alternatively, we could rework this a bit and make one function that combines both getConversations and getEvaluations and results in a data structure that is an array of
{
convo: Conversation,
evals: Evaluation[]
}

This would reduce the need to constantly check for uniqueness in the conversations array and it would be a bit more clear when we're in a data fetch state.  Granted, the conversations won't come in as they're resolved, but I'm not sure that's necessary.

This is just a bug fix to an existing impl tho, so I'm good either way.